roles/kubernetes-apps/scheduler_plugins/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+scheduler_plugins_enabled: false
+
+scheduler_plugins_controller_replicas: 1
+
+scheduler_plugins_scheduler_replicas: 1
+# The default is determined by the number of control plane nodes.
+scheduler_plugins_scheduler_leader_elect: "{{ ((groups['kube_control_plane'] | length) > 1) }}"
+
+# Plugins to enable. See https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/manifests/install/charts/as-a-second-scheduler/README.md#configuration for more info.
+# scheduler_plugins_enabled_plugins:
+#   - Coscheduling
+#   - CapacityScheduling
+#   - NodeResourceTopologyMatch
+#   - NodeResourcesAllocatable
+
+# Plugins to disable. See https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/manifests/install/charts/as-a-second-scheduler/README.md#configuration for more info.
+# scheduler_plugins_disabled_plugins:
+#   - PrioritySort
+
```

Comment:
HI @tu1h 

How about uncomment the `scheduler_plugins_enabled_plugins` and `scheduler_plugins_disabled_plugins`, and using the default value like : https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/manifests/install/charts/as-a-second-scheduler/values.yaml#L20